documentation/src/docs/streamCuts.md

```diff
@@ -0,0 +1,90 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Working with Pravega: StreamCuts
+
+This section describes about `StreamCut`s and how it can be used with streaming clients and batch clients.
+Pre-requisites: You should be familiar with Pravega Concepts (see http://pravega.io/docs/latest/pravega-concepts/).
```

Comment:
In other parts of the documentation the link is embedded in the words "Pravega Concepts" (i.e., [Pravega Concepts](http://pravega.io/docs/latest/pravega-concepts/)).

Reply:
done.